index.html

```diff
@@ -44,6 +44,158 @@ <h2 class="name1">I’am a Software Developer</h2>
+         <br>
+       <div class="tags1">
+         <ul>
+           <li class="tag html1">html</li>
+           <li class="tag bootstrap1">bootstrap</li>
+           <li class="tag ruby1">Ruby</li>
+         </ul>
+       </div>
+         <br>
+         <button class="projects1"><span class="see-projects1">See project</span></button>
+     </div>
+     </div>
+
+     <div class="showcaseQ">
+       <div class="Art">
+         <br><br><br><br>
+         <h2 class="stories1"> Profesional Art Printing Data</h2>
+         <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         <br>
+         <div class="tags1">
+           <ul>
+             <li class="tag html1">html</li>
+             <li class="tag bootstrap1">bootstrap</li>
+             <li class="tag ruby1">Ruby</li>
+           </ul>
+         </div>
+           <br>
+           <button class="projects1"><span class="see-projects1">See project</span></button>
+       </div>
+       </div>
+   </section>
+   <section class="about">
+       <h2 class="myself">About me</h2>
+       <p id="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+       <button class="get"><span class="resume">Get My Resume</span></button>
+       <img class="pics" src="img/illustration-about-me-mobile.png" alt="">
+       <li class="barQk"><div id="barQk"></div></li>
+
+    <article>
+      <div>
+       <li class="lang"><img class="image" src="img/Rectangle 55@3x.png" alt="">Language</li>
+          <ul id="list">
+            <li class="lang1">Javascript</li>
+            <li class="lang2">Ruby</li>
+            <li class="lang3">Html</li>
+            <li class="lang4">Css</li>
+          </ul>  
+      </div>
+    </article>
+     <br><br><br><br><br><br><br><br><br>
+    <article>
+       <div>
+         <li class="langv"> <img class="image4" src="img/Rectangle 55@3x.png" alt="">Framework</li>
+          <ul id="list">
+            <li class="lang1">Bootstrap</li>
+            <li class="lang2">Ruby on Rails</li>
+            <li class="lang3">RSPec</li>
+            <li class="lang4">CapyBara</li>
+            <li class="lang4">Selenium</li>
+          </ul>  
+       </div>
+   </article>
+     <br><br><br><br><br><br><br><br><br><br><br><br><br>
+   <article>
+      <div>
+       <li class="langvi"> <img class="image5" src="img/Rectangle 57.png" alt="">Skills</li>
+         <ul id="list">
+           <li class="lang1">Codekit</li>
+           <li class="lang2">Github</li>
+           <li class="lang3">CodePen</li>
+           <li class="lang4">Gitlab</li>
+           <li class="lang4">Terminal</li>
+         </ul>  
+       </div>
+   </article>
```

Comment:
Good job using a `li` tag for `languages, framework, skills` section. For semantic reasons, when using `li` tags, it is important to group them in a `ul` tag. Instead of wrapping the individual `li` tags for  `languages, framework, skills` section in an `article` and `div` tags, wrap them in one `ul` tag 👍 👍